node/core/dispute-coordinator/src/scraping/candidates.rs

```diff
@@ -143,7 +143,7 @@ impl ScrapedCandidates {
 		block_hash: Hash,
 		candidate_hash: CandidateHash,
```

Comment:
nit: This `insert` can also be fixed so that the key is first, and to match the other one. 🙂